README.md

```diff
@@ -1,6 +1,6 @@
-# Todo List with Interactivity
+# Todo List with Add and Remove function
 
-It is a Todo list with CURD (create, update, delete) methods  and the user will also be able to mark task completion by selecting the corresponding checkbox. The updated tasks list will be stored in local storage.
+- This project is a todo list project with add and remove function
 
```

Comment:
- [ ] it would be better if the first task should have 1 not 0 on the index and also when I delete a task the index should be updated as required as you can see in the screenshot your first task has 0 on the index 
- Screenshot of the requirements 
<img width="1229" alt="Screen Shot 2022-11-03 at 16 44 52" src="https://user-images.githubusercontent.com/61790587/199767709-37f53e3b-0b1d-41b4-874c-2e82b55646cc.png">
- Your app 
<img width="1436" alt="Screen Shot 2022-11-03 at 16 42 17" src="https://user-images.githubusercontent.com/61790587/199767744-a6bc3ea0-cc8c-4398-8e0a-dd652e4c8600.png">